diff --git a/openxr-api-layer/ProjectionPS.hlsl b/openxr-api-layer/ProjectionPS.hlsl
--- a/openxr-api-layer/ProjectionPS.hlsl
+++ b/openxr-api-layer/ProjectionPS.hlsl
@@ -66,7 +66,7 @@ float4 main(in float4 position : SV_POSITION, in float2 texcoord : PROJ_COORD0, in f
     if (smoothingArea) {
         float2 s = smoothstep(float2(0, 0), float2(smoothingArea, smoothingArea), layer1TexCoord) -
                    smoothstep(float2(1, 1) - float2(smoothingArea, smoothingArea), float2(1, 1), layer1TexCoord);
-        color1.a = isInside * max(0.5, s.x * s.y);
+        color1.a = isInside * max(0.0, s.x * s.y);
     } else {
         color1.a = isInside;
     }
